Chat/appsettings.json

```diff
@@ -1,11 +1,13 @@
 {
-    "Logging": {
-        "LogLevel": {
-            "Default": "Trace",
-            "System": "Information",
-            "Microsoft": "Information"
-        }
-    },
-    "AllowedHosts": "*",
-    "ResourceConnectionString": "REPLACE_WITH_CONNECTION_STRING"
+  "Logging": {
+    "LogLevel": {
+      "Default": "Trace",
+      "System": "Information",
+      "Microsoft": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "ResourceConnectionString": "REPLACE_WITH_CONNECTION_STRING",
```

Comment:
Let's add a config to enable and disable the button? Could be either here or next PR, up to you

Reply:
yes we are planning of adding that in another PR.